### DISCUSSION TITLE
Support to markdown field plugin

### DISCUSSION THREAD

**Discussion by zebomfim** [kirby-tools/community#27] [Q&A] [answered]
Hi, thanks for the plugin! I've started testing it in my local environment, but it doesn't seem to work with the [Markdown plugin by S. Julé and F. Michael](https://github.com/fabianmichael/kirby-markdown-field). Is this a known issue, or am I missing something?

**Comment by johannschopplich** [comment]
Hi @zebomfim! Which plugin are you talking about, Kirby Copilot or Content Translator?

**Comment by zebomfim** [comment]
Kirby content translator, I believe that is the same doubt:
https://github.com/kirby-tools/kirby-content-translator/discussions/4

--
*José Luis Bomfim - +34 678 332 173*

**Reply by johannschopplich** [reply, marked as answer]
You're right! It was about time to support `type: markdown` for content translation.

Please upgrade to [Kirby Content Translator v2.5.0](https://kirby.tools/docs/content-translator/changelog#whats-new) for added markdown field support.

**Reply by zebomfim** [reply]
Thanks!
